commercetools-models/src/main/java/io/sphere/sdk/categories/commands/updateactions/SetAssetCustomField.java

```diff
@@ -0,0 +1,32 @@
+package io.sphere.sdk.categories.commands.updateactions;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.types.customupdateactions.SetCustomFieldBase;
+
+/**
+ * This action sets, overwrites or removes any existing custom field for an existing Asset.
+ *
+ * @see io.sphere.sdk.types.Custom
+ */
+public final class SetAssetCustomField extends SetCustomFieldBase<Category> {
+    private final String assetId;
+
+    private SetAssetCustomField(final String assetId, final String name, final JsonNode value) {
+        super("setAssetCustomField", name, value);
+        this.assetId = assetId;
+    }
+
+    public String getAssetId() {
+        return assetId;
+    }
+
+    public static SetAssetCustomField usingJson(final String assetId, final String name, final JsonNode value) {
```

Comment:
Now I'm checking the same with products, I guess you removed it because it sounded weird. Let's see... I would just simply say `ofJsonValue`, sounds better than `ofUsingJson` or `ofJson` (which is misleading). The "using" part from the other classes forces us to change a bit the structure here, which is bad, but it's the best option I can think of and yet it's easy to understand.